src/new/wallet.js

```diff
@@ -351,7 +473,13 @@ class HathorWallet extends EventEmitter {
       address = wallet.getCurrentAddress();
     }
     const index = this.getAddressIndex(address);
-    const addressPath = `m/44'/${HATHOR_BIP44_CODE}'/0'/0/${index}`;
+    let addressPath;
+    if (wallet.isWalletMultiSig()) {
+      addressPath = `m/45'/${HATHOR_BIP44_CODE}'/0'/0/${index}`;
```

Comment:
Should we use the var in the constants file?

Reply:
Done in [89d3a89](https://github.com/HathorNetwork/hathor-wallet-lib/pull/324/commits/89d3a89fb831ed94006445e3d29178170df31c29)